config.json

```
@@ -2465,6 +2465,20 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "a68113f8-92be-40e0-a97f-eb4239303ef5",
+        "prerequisites": [
+          "cond",
+          "case",
+          "if"
+        ],
+        "practices": [
+          "cond"
+        ],
```

Ok, if we're leaving out erlang-libraries, I would want this to be a very early exercise which we can achieve by deciding that you're supposed to use `cond`, not `case` or `if or `multiple-clause-functions`
```suggestion
        "prerequisites": [
          "cond",
          "integers",
          "floating-point-numbers",
        ],
        "practices": [
          "cond"
        ],
```


I think it's a reasonable suggestion. It looks more crowded with `case`/`multiple-clause-functions` and `guards`:
```elixir
    cond do
      radius > 10 -> 0
      radius > 5 -> 1
      radius > 1 -> 5
      true -> 10
    end
```
```elixir
    case radius do
      radius when radius > 10 -> 0
      radius when radius > 5 -> 1
      radius when radius > 1 -> 5
      radius -> 10
    end
```
```elixir
    defp do_score(radius) when radius > 10, do: 0
    defp do_score(radius) when radius > 5, do: 1
    defp do_score(radius) when radius > 1, do: 5
    defp do_score(radius), do: 10
```

---

Sounds good to me